python/gigl/common/data/dataloaders.py

```diff
@@ -356,11 +368,16 @@ def load_as_torch_tensors(
                 if entity_type == FeatureTypes.NODE
                 else torch.empty(2, 0)
             )
-            empty_feature = (
-                torch.empty(0, serialized_tf_record_info.feature_dim)
-                if feature_keys
-                else None
-            )
+            if label_keys and feature_keys:
+                empty_feature = torch.empty(
+                    0, serialized_tf_record_info.feature_dim + len(label_keys)
+                )
+            elif label_keys and not feature_keys:
+                empty_feature = torch.empty(0, len(label_keys))
+            elif not label_keys and feature_keys:
+                empty_feature = torch.empty(0, serialized_tf_record_info.feature_dim)
+            else:
+                empty_feature = None
```

Comment:
oops sorry to go back again but WDYT about 
```suggestion
            if feature_iterable:
                empty_feature = torch.empty(
                    0, len(feature_iterable)                )
            else:
                empty_feature = None
```

Reply:
This would not necessarily work -- `len(feature_iterable) = len(feature_keys) + len(label_keys)`.  `len(feature_keys)` and `serialized_tf_record_info.feature_dim` are not necessarily the same, since we could have one feature key with dim > 1